pkg/apis/core/v1alpha1/types_common.go

```diff
@@ -108,6 +108,8 @@ type Gardener struct {
 	Name string `json:"name" protobuf:"bytes,2,opt,name=name"`
 	// Version is the version of the Gardener which last acted on a resource.
 	Version string `json:"version" protobuf:"bytes,3,opt,name=version"`
+	// ClusterIdentity is the cluster identity of the Gardener cluster
+	ClusterIdentity string `json:"clusterIdentity" protobuf:"bytes,4,opt,name=clusterIdentity"`
```

Comment:
Why is it added here? It's even not added as an optional field but as required field which violates the API conventions. Now it will be written to every shoot's `.status.gardener`, so there will be `.status.gardener.clusterIdentity` and `.status.clusterIdentity` which is highly confusing.
I suggest to remove this field from the API and simply handle it internally in the Gardenlet code.

Reply:
Okay, will change it